### ISSUE TITLE
Recursion loop in symfony v4.2.2 using symfony bundle

### ISSUE THREAD

**Issue by aphex008** [php-enqueue/enqueue-dev#741]
In symfony v4.2.2 when processor encounters exception, exception is outputed in console in recursion. Console is not responding to input. If monolog is used, app crashes because uses up all memory. It appears that exception handler (\Enqueue\Consumption\QueueConsumer::onProcessorException) injecs exception to it self as previous exception and creates recursion. 

Tried it on my app. Also tried it on clean symfony install. See detail steps to reproduce below.

Steps to reproduce:
1. ```bash composer create-project symfony/skeleton my-project```
2. ```bash composer require enqueue/enqueue-bundle```
3. ```bash composer req enqueue/amqp-lib```
4. Change .env ENQUEUE_DSN=amqp+lib:
5. Create basic processor my-project/src/Processor/PrimaImportProcessor.php:

```php
<?php declare(strict_types = 1);

namespace App\Processor;

use Interop\Queue\Context;
use Interop\Queue\Message;
use Interop\Queue\Processor;

class PrimaImportProcessor implements Processor
{
    public function process(Message $message, Context $context)
    {
        throw new \RuntimeException('a');
    }
}
```
6. Define it in config/services.yaml 
```yaml

services:
    # default configuration for services in *this* file
    _defaults:
        autowire: true      # Automatically injects dependencies in your services.
        autoconfigure: true # Automatically registers your services as commands, event subscribers, etc.

    # makes classes in src/ available to be used as services
    # this creates a service per class whose id is the fully-qualified class name
    App\:
        resource: '../src/*'
        exclude: '../src/{DependencyInjection,Entity,Migrations,Tests,Kernel.php}'

    # controllers are imported separately to make sure services can be injected
    # as action arguments even if you don't extend any base controller class
    App\Controller\:
        resource: '../src/Controller'
        tags: ['controller.service_arguments']

    # add more service definitions when explicit configuration is needed
    # please note that last definitions always *replace* previous ones


    App\Processor\PrimaImportProcessor:
        public: true
        tags:
            - { name: 'enqueue.transport.processor', processor: 'prima_import_processor' }
```
7. Run processor
```bash
console enqueue:transport:consume prima_import_processor edss-import -vvv
```

8. Get eternal loop in console
https://i.imgur.com/I28dObw.png

![image](https://user-images.githubusercontent.com/4013304/51854910-99f8c600-2334-11e9-8388-a91fd2ec390b.png)

My composer json is
```json
{
    "type": "project",
    "license": "proprietary",
    "require": {
        "php": "^7.1.3",
        "ext-ctype": "*",
        "ext-iconv": "*",
        "enqueue/amqp-lib": "^0.9.2",
        "enqueue/enqueue-bundle": "^0.9.3",
        "symfony/console": "4.2.*",
        "symfony/dotenv": "4.2.*",
        "symfony/flex": "^1.1",
        "symfony/framework-bundle": "4.2.*",
        "symfony/yaml": "4.2.*"
    },
    "config": {
        "preferred-install": {
            "*": "dist"
        },
        "sort-packages": true
    },
    "autoload": {
        "psr-4": {
            "App\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "App\\Tests\\": "tests/"
        }
    },
    "replace": {
        "paragonie/random_compat": "2.*",
        "symfony/polyfill-ctype": "*",
        "symfony/polyfill-iconv": "*",
        "symfony/polyfill-php71": "*",
        "symfony/polyfill-php70": "*",
        "symfony/polyfill-php56": "*"
    },
    "scripts": {
        "auto-scripts": {
            "cache:clear": "symfony-cmd",
            "assets:install %PUBLIC_DIR%": "symfony-cmd"
        },
        "post-install-cmd": [
            "@auto-scripts"
        ],
        "post-update-cmd": [
            "@auto-scripts"
        ]
    },
    "conflict": {
        "symfony/symfony": "*"
    },
    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "4.2.*"
        }
    }
}
```

**Comment by aphex008** [comment]
BTW works fine on symfony 3.4.

**Comment by aphex008** [comment]
Php version
PHP 7.2.13 (cli) (built: Dec  7 2018 10:41:23) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.2.0, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.2.13, Copyright (c) 1999-2018, by Zend Technologies

**Comment by DamienHarper** [comment]
@aphex008  it looks like what have been fixed in #725, could you test against `master` branch?

**Comment by makasim** [comment]
> @aphex008 it looks like what have been fixed in #725, could you test against master branch?

If so, should be fixed in 0.9.7 `
